content/en/docs/step-by-step/2-robot-modification/robot-specific-guides/Sabo/index.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "SABO / John Deere"
+linkTitle: "SABO / John Deere"
+weight: 20
+description: >
+  Modification Guide for SABO MOWiT 500F / John Deere Tango E5
+---
+{{% alert title="v2 Hardware Only!" color="warning" %}}
+This guide here is for OpenMower v2.x Hardware only!
+{{% /alert %}}
```

Comment:
We can remove that warning because there's no v1 Hardware for SABO Platform

Reply:
Yes